app/scripts/controllers/preferences-controller.ts

```diff
@@ -127,15 +127,18 @@ export type Preferences = {
 // Omitting showTestNetworks and smartTransactionsOptInStatus, as they already exists here in Preferences type
```

Comment:
Nit: Maybe we could update this comment to remove the list of specific properties omitted (since it would be too long now, and it's evident from the code). We could also specify that they're all on the `preferences` property

Reply:
updated to
```
// Omitting properties that already exist in the PreferencesState, as part of the preferences property.
```